core/src/card.ts

```diff
@@ -0,0 +1,13 @@
+import { MonetaryValue } from "./monetaryvalue";
+
+export class Card {
+  level: number;
+  cost: MonetaryValue;
+  purchasingPower: MonetaryValue;
+
+  constructor(level: number, cost: MonetaryValue, purchasingPower: MonetaryValue) {
+    this.level = level;
+    this.cost = cost;
+    this.purchasingPower = purchasingPower;
+  }
```

Comment:
Do you think there are other properties / functions that Card needs?

Reply:
I think we might be missing a `points` field?

Reply:
[fixed](https://github.com/qoosuperman/panoply/pull/5/commits/c86e38d727457f4d74930d680dd08f5c0f35868d)